dll/win32/shell32/wine/shellord.c

```diff
@@ -1388,8 +1388,15 @@ HRESULT WINAPI SHWinHelp(HWND hwnd, LPCWSTR pszHelp, UINT uCommand, ULONG_PTR dw
  */
 BOOL WINAPI SHRunControlPanel (LPCWSTR commandLine, HWND parent)
 {
+#ifdef __REACTOS__
```

Comment:
Is this needed, as ROS shell32 is forked?

Reply:
There are __REACTOS__ protection is other place of shell32. Let's discuss in the Dev channel to see if there is a global policy on use of __REACTOS__ on this forked component.